src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js

```diff
@@ -132,6 +132,7 @@ const MappingProfilesContainer = ({
                 {...editPageProps}
                 allTransformations={allTransformations}
                 onCancel={handleEditPageCancel}
+                onSave={handleNavigationToMappingProfilesList}
```

Comment:
I suggest handling the logic here exactly like [it is done|https://github.com/folio-org/ui-data-export/blob/v3.0.0/src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js#L164..L175] for the duplicate mapping profile feature.

Reply:
Do you mean naming the prop?

Reply:
I am thinking about having two props like there for consistency.
```
                onCancel={handleNavigationToMappingProfilesList}
                onSubmitNavigate={handleNavigationToMappingProfilesList}
```